spring-cloud-aws-app-config/src/main/java/io/awspring/cloud/appconfig/RequestContext.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2013-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.awspring.cloud.appconfig;
+
+/**
+ * @author Matej Nedic
+ * @since 3.0
+ */
+public class RequestContext {
```

Comment:
From AppConfig perspective, there is known attributes such as `configurationProfileIdentifier`, `environmentIdentifier` and `applicationIdentifier`.  However, `context` is something to be used internally AFAICS. I would suggest to use and internal format to be built once the other attributes have been filled.

Reply:
So we should pass `String context` which is full path and in `AppConfigPropertySource` split it up to `configurationProfileIdentifier`, `environmentIdentifier` and `applicationIdentifier`? If I understood it right.